start-new-game.md

```diff
@@ -1,3 +1,7 @@
-I'm normal, so [I choose you](english/normal-dungeon/0/0.md)
+# Welcome to the Dungeon!
 
-[不到長城非好漢](chinese/great-wall/0/0.md)
\ No newline at end of file
+Choose your language:
```

Comment:
This will be a list of the dungeon, so it is not the language list.
It could choose the dungeon.

Reply:
Alright, maybe choose your dungeon?

Reply:
Nice